packages/plugin-infra/src/utils/createInjectHooksRenderer.tsx

```diff
@@ -43,15 +43,15 @@ export function createInjectHooksRenderer<PluginDefinition extends Plugin.Shared
         )
     }
     function PluginsInjectionHookRender(props: PropsType) {
-        const all = usePlugins()
-            .filter(pickInjectorHook)
-            .map((plugin) => (
-                <PropsContext.Provider key={plugin.ID} value={props}>
-                    <ShadowRootIsolation data-plugin={plugin.ID}>
-                        <SinglePluginWithinErrorBoundary plugin={plugin} />
-                    </ShadowRootIsolation>
-                </PropsContext.Provider>
-            ))
+        const allPlugins = usePlugins()
```

Comment:
this change will affect every plugin injection point. Please make sure you really want to do this.

Reply:
Yes, to hide next id on non-EVM chains.

Reply:
if you only want to hide NextID, you should not change here because it will affect _every plugin_

Reply:
Not just NextID, but also other plugins that match the rule set by `enableRequirement.web3`.